src/ExpressiveInstaller/Resources/config/container-auryn.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+use Auryn\Injector;
+use Northwoods\Container\InjectorContainer;
+use Psr\Container\ContainerInterface;
+
+// Load configuration
+$config = require __DIR__ . '/config.php';
+
+// Create injector
+$injector = new Injector();
+
+// Configuration will be used globally
+$injector->share('config')->delegate('config', function () use ($config) {
+    // Auryn requires that all injections resolve to an object.
+    // Wrap the config array with an ArrayObject to appease the gods.
```

Comment:
@Ocramius Aura.DI still requires an object. I just moved it to the Aura.DI container file as @shadowhand linked.